mk/cert-manager.mk

```diff
@@ -1,22 +1,12 @@
 
-CERT_MANAGER_CVS_NAME=$(shell kubectl get Subscription/cert-manager -n openshift-operators -o jsonpath='{.status.installedCSV}')
+##@ Cert Manager
 
 .PHONY: cert-manager-install
-cert-manager-install:
+cert-manager-install:  ## Install cert-manager operator
 	kubectl create -f config/certmanager/subscription.yaml
+	kubectl wait Subscription/cert-manager -n openshift-operators --for=condition=CatalogSourcesUnhealthy=False
```

Comment:
We need cert-manager operator when we deploy the controller in our development OpenShift cluster without OLM by doing 'make deploy'.